public/pages/AnomalyCharts/containers/AnomaliesChart.tsx

```diff
@@ -84,6 +84,8 @@ interface AnomaliesChartProps {
   onZoomRangeChange(startDate: number, endDate: number): void;
   title: string;
   anomalies: any[];
+  atomicAnomalies: boolean;
```

Comment:
what does this mean?

Reply:
If user choose a long time range, we will show bucketized anomalies. If the date range is short enough to show every AD data point, will set `atomicAnomalies` as true

Reply:
I see. Maybe you can change its name to "showBucketizedAnomalies". `atomic` sounds a little bit confusing to me.

Reply:
Done. changed to `bucktizedAnomalies`